src/transformers/modeling_bart.py

```diff
@@ -1110,8 +1110,7 @@ def forward(
 
         masked_lm_loss = None
         if labels is not None:
-            loss_fct = CrossEntropyLoss()
-            # TODO(SS): do we need to ignore pad tokens in labels?
+            loss_fct = CrossEntropyLoss(ignore_index=self.config.pad_token_id)
```

Comment:
`assert` if there are -100 in labels

Wdyt @sshleifer

Reply:
Great idea. We should also probably do `FSMT`

Reply:
Good point, I remember that through me off because it explicitly says -100 works in the model's [docstring](https://github.com/huggingface/transformers/blob/4dbca500226e27be21dbb0eb08117dfd0c5264b3/src/transformers/modeling_bart.py#L1048).